### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by MikaelMayer**
<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

**Review by cpitclaudel — CHANGES_REQUESTED** [review]
I think that's not right: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates says ISSUE_TEMPLATE should be a directory.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - type: dropdown
+    id: dafnyversion
+    attributes:
+      label: What version of Dafny were you using?
+      multiple: false
+      options:
+        - latest
+        - 3.9.X
+        - 3.8.X
+        - 3.7.X
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:21]: How about making this freeform, so we don't have to keep updating it?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        If you are having trouble with a proof?  Consider posting your question on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=dafny).	  
+        Please do not report security issues on this public tracker.  Instead, see https://github.com/dafny-lang/dafny/blob/master/.github/SECURITY.md
+  - type: dropdown
+    id: dafnyversion
+    attributes:
+      label: What version of Dafny were you using?
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:15]: Maybe we could explain how to find out the version

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        If you are having trouble with a proof?  Consider posting your question on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=dafny).
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:10]:
```suggestion
        Are you having trouble with a proof?  Consider posting your question on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=dafny).	  
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: []
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:4]: `kind: bug` here?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        - 3.9.X
+        - 3.8.X
+        - 3.7.X
+        - other
+    validations:
+      required: true
+  - type: textarea
+    id: code
+    attributes:
+      label: Code to produce this issue.
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:28]: No periods here I think (these are titles IIRC)

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      description: Please paste a minimal example that reproduces the issue.
+      render: dafny
+  - type: textarea
+    id: environment
+    attributes:
+      label: Command to run into the issue or logs
+      description: "Please paste here what you ran and what were your your logs that you think ought to be different:"
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:35]: What's a log?

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+name: Feature Request
+description: Request a feature for Dafny
+title: "[Feature request]: "
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to give us ideas on how to evolve the Dafny language!
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:9]: It's not just that, thought, right?

**Review by cpitclaudel — CHANGES_REQUESTED** [review]
Looks  great!  Should we add another template for our own tasks (since we use the repo a lot for that), and maybe a freeform option as well?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml (hunk 4 above)
Comment (MikaelMayer) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:4]:
```suggestion
labels: []
kind: bug
```

**Comment by MikaelMayer** [comment]
> Looks great! Should we add another template for our own tasks (since we use the repo a lot for that), and maybe a freeform option as well?

I think there is always a freeform option available, so we could use it for our own tasks for now. Let's do this iteratively.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml (hunk 4 above)
Comment (MikaelMayer) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:4]: kind is not accepted

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+      label: What type of Operating System are you seeing the problem on?
+      multiple: true
+      options:
+        - Linux
+        - Windows
+        - Mac
+        - Other
+    validations:
+      required: true
+
```
Comment (fabiomadge) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:52]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+        Thanks for your feature request!
+  - type: textarea
+    id: what-do-you-want
+    attributes:
+      label: What is the feature you would like to see in a future version of Dafny?
+      description: Describe the feature here. Markdown is supported.
+      placeholder: I would like to have X
+    validations:
+      required: true
+
```
Comment (fabiomadge) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:18]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-systems
+    attributes:
+      label: What type of Operating System are you seeing the problem on?
+      multiple: true
+      options:
+        - Linux
```
Comment (cpitclaudel) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:46]: Maybe we want to add an option for WSL?

**Review by cpitclaudel — DISMISSED** [review]

**Review by cpitclaudel — APPROVED** [review]
👏

**Pull request merged**